apps/test-app/frontend/src/components/tree-widget/TreeWidget.tsx

```diff
@@ -32,16 +33,50 @@ import { Tree } from "./Tree";
 interface Props {
   imodel: IModelConnection;
   rulesetId?: string;
+  height?: number;
 }
 
 export function TreeWidget(props: Props) {
+  const [openTab, setOpenTab] = useState(0);
+  const { height, ref } = useResizeDetector();
+  const tabsClassName = "tree-widget-tabs";
+  const [heightOfTreeWidget, setHeightOfTreeWidget] = useState(0);
+  const tabElements = document.getElementsByClassName(tabsClassName);
```

Comment:
Don't use this to access native html element. Use `useRef(null)` instead.

Reply:
`<Tabs>`  and `<Tab>` elements don't have `ref` property, so `useRef` can't be used with them. You can set useRef for each `<div className="filteredTree">`, but then, when trying to resize the right panel, "filteredTree" element height gets updated very slowly.
Changed to:
`ref.current?.getElementsByClassName(tabsClassName);` and moved it to the `useEffect` block.